tinynn/graph/quantization/quantizable/gru.py

```diff
@@ -0,0 +1,341 @@
+from distutils.version import LooseVersion
+
+import numbers
+from typing import Optional, Tuple
+import warnings
+
+import torch
+from torch import Tensor
+
+if LooseVersion(torch.__version__) >= '1.13.0':
+
+    @classmethod
+    def from_float(cls, other, qconfig=None):
+        assert isinstance(other, cls._FLOAT_MODULE)
+        assert hasattr(other, 'qconfig') or qconfig
+        observed = cls(
+            other.input_size,
+            other.hidden_size,
+            other.num_layers,
+            other.bias,
+            other.batch_first,
+            other.dropout,
+            other.bidirectional,
+        )
+        observed.qconfig = getattr(other, 'qconfig', qconfig)
+        for idx in range(other.num_layers):
+            observed.layers[idx] = _GRULayer.from_float(other, idx, qconfig, batch_first=False)
+        observed.train()
+        observed = torch.ao.quantization.prepare_qat(observed, inplace=True)
+        return observed
+
+
+class GRUCell(torch.nn.Module):
```

Comment:
It seems that those classes are not wrapped with the `if LooseVersion(torch.__version__) >= '1.13.0':` protection, please make sure those can run with PyTorch 1.6.